README.md

```diff
@@ -79,30 +80,6 @@ try {
 
 Upon calling `startTransaction()`, a transaction will be started with the card related to this connection. When successful, the given callback will be called. All interactions with the card done inside this callback will then be part of this transaction. The transaction is ended once the `Promise` returned by the callback settles.
 
-## Reacting to reader availability
```

Comment:
This change makes it look like developers can't get reader add/remove events. I don't think this is true. Can you replace this with an example that works with the low-level interface?

Reply:
I didn't include a new example for this here mainly because it will be the standard PC/SC fare. And it won't be pretty. :)
But given `GetStatusChange` is the most awkward part of PC/SC, it's indeed worth showing a snippet on how to deal with it in its Web incarnation.